src/Restyler/App/Error.hs

```diff
@@ -4,14 +4,22 @@ module Restyler.App.Error
     ( AppError(..)
     , mapAppError
     , prettyAppError
+
+    -- * Error handling
```

Comment:
TODO: these are now good cases to try out testing `RIO` actions in a test `env`. Ensuring we don't over-handle `ExitCode` exceptions is an important thing to regression-cover.